src/Note.vue

```diff
@@ -143,7 +147,14 @@ export default {
 }
 </script>
 <style scoped>
+.note-wrapper {
+	height: calc(100vh - 50px);
```

Comment:
This will cause issues in all Apple related devices & desktops because strangely, safari does not count thenavigation bar against the 100vh, so even with 100vh, it will still scroll vertically (no comment :roll_eyes:) You should use the 100% we implemented on nc14 if possible :)

What is this note-editor element? A modal?

Reply:
Thanks for the hint. I will fix this.

`note-editor` is not a modal element. It fills the whole `app-content`with 100%. It contains the note's editor which is a better textarea with live markdown preview (like in the current version of the notes app).

Reply:
By the way: thanks for reviewing this :-)

Reply:
> `note-editor` is not a modal element. It fills the whole `app-content`with 100%. It contains the note's editor which is a better textarea with live markdown preview (like in the current version of the notes app).

Then I'll suggest you use the modal component with the `'full'` size. We want to unify the look of modal sin nextcloud and you will benefit from any future updates as well ;)

https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/Modal/Modal.vue